src/pages/blog.jsx

```diff
@@ -0,0 +1,113 @@
+import PageIntro from "@/components/PageIntro";
+import Image from "next/image";
+import { db, storage } from "@/firebase/config";
```

Comment:
we have build reusable addDate no need to us this , please use import addDocument from "@/firebase/addData";

Reply:
already use it ,done